client/tracks/index.js

```diff
@@ -31,10 +31,10 @@ function recordEvent( eventName, eventProperties ) {
 }
 
 /**
- * Records events from buyers.
+ * Records events from buyers (aka shoppers).
  *
- * @param {string}  eventName       Name of the event.
- * @param {Object?} eventProperties Event properties.
+ * @param {string}  eventName       	Name of the event.
+ * @param {Object}  [eventProperties] Event properties (optional).
```

Comment:
@Jinksi is it a good idea for us to rename this to .ts now and fix things up? Or is that a dangerous thread to pull?

Reply:
I know this has already been merged, but I'd say pull that thread! The biggest risk is that it would be a rabbithole of updates to files downstream.

This might be good as an individual PR.

Reply:
> I know this has already been merged, but I'd say pull that thread! The biggest risk is that it would be a rabbithole of updates to files downstream.

I think it was pretty minor, since we import module name (no extension). Will give it a go when I get a chance!